### PR TITLE
Added Dotenv

### PR DESCRIPTION
**Pull request by charlie1404**
Added Dotenv
Modified instruction to set up in README.md.

### CODE REVIEW

**Comment by VarunDevPro** [comment]
This is really a good one, but will the cp command work on all the OS, if not try to add support for all the OS.

**Comment by skyshader** [comment]
Looks good to me. Have you verified if things are working as expected? @charlie1404

**Comment by VarunDevPro** [comment]
Ok Since I work only with Windows I don't know whether those commands work similarly on Mac and Linux. I have seen that copy is used instead of cp for Windows. 

I have also seen that some commands which are not recognised in **Command Prompt** can be recognised by **git bash** on Windows. 
If this is the case with **cp**, we should tell the users clearly when to use what.

File: server/server.js [hunk 1]
```diff
@@ -1,3 +1,8 @@
+const envVars = require("dotenv").config();
```
[the remark this answers is not in the document]
Comment (VarunDevPro) [reply server/server.js]: Yes it will good have simpler name.

File: server/.gitignore [hunk 2]
```diff
@@ -11,3 +11,8 @@ yarn-error.log*
 *.ntvs*
 *.njsproj
 *.sln
+
+.env
+.env.example
```
[the remark this answers is not in the document]
Comment (VarunDevPro) [reply server/.gitignore]: Yes, users should know how to configure their env file so `env.example` should not be ignored.

**Comment by charlie1404** [comment]
I have added .env.example to gitignore but project is using env.example, as in linux or similar os'es it is hidden, but env.example will be visible in file explorer
though I have removed that from gitignore, and changed envVars to env only

**Comment by skyshader** [comment]
@charlie1404 can you submit the PR to `develop` branch as we have moved our work on that. If I merge here, it might cause a lot of merge issues.

**Comment by charlie1404** [comment]
How do i Move this Pull request to develop branch, Please help

**Comment by charlie1404** [comment]
Closing this pull request in favor of pull #48

**Pull request closed without merging**